### PR TITLE
Move admin user filters to GQL

### LINKED ISSUE #1108: Admin dashboard: User role and guest filters should be server-side
**Describe the bug**
On the admin dashboard the new user role and guest filters were all implemented client-side.
That doesn't work, because we don't fetch all the users.

**To Reproduce**
1. Go to https://languagedepot.org/admin?userType=admin
2. The table says there are no Admins

### PR DESCRIPTION
**Pull request by myieye**
Resolves #1108

### CODE REVIEW

File: frontend/src/routes/(authenticated)/admin/+page.ts [hunk 1]
```diff
@@ -139,6 +135,37 @@ export async function load(event: PageLoadEvent) {
...
+        ]
+      };
+  const usersICreatedFilter: UserFilterInput =
+    onlyUsersICreated ? {createdById: {eq: user.id as UUID}}
+      : {};
+  const userTypeFilter: UserFilterInput =
+    userType === 'admin' ? {isAdmin: {eq: true}}
+      : userType === 'nonAdmin' ? {isAdmin: {eq: false}}
+        : userType === 'guest' ? {createdById: {neq: null}}
+          : {};
```
Comment (hahn-kev) [inline frontend/src/routes/(authenticated)/admin/+page.ts:160]: could this be a switch statement instead?

File: frontend/src/routes/(authenticated)/admin/+page.ts [hunk 2]
```diff
@@ -75,7 +73,11 @@ export async function load(event: PageLoadEvent) {
               userCount
             }
             draftProjects(
-              where: $draftFilter
+              where: $draftFilter,
+              orderBy: [
```
Comment (hahn-kev) [inline frontend/src/routes/(authenticated)/admin/+page.ts:77]: I snuck this change in as I just noticed it today...

**Review by hahn-kev — DISMISSED** [review]
nice job!

**Comment by myieye** [comment]
* **#1219** <a href="https://app.graphite.dev/github/pr/sillsdev/languageforge-lexbox/1219?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @myieye and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: frontend/src/routes/(authenticated)/admin/+page.ts (hunk 1 above)
Comment (myieye) [reply frontend/src/routes/(authenticated)/admin/+page.ts]: I rewrote it. Among other things, I used a switch for this.

**Comment by myieye** [comment]
Addressed all feedback and actually fixed a minor bug

**Pull request merged**